tests/integration/did.test.js

```diff
@@ -59,6 +60,22 @@ describe('DID Module', () => {
     expect(!!result).toBe(true);
   }, 10000);
 
+  test('Can attest with a DID', async () => {
```

Comment:
A few more tests I think belong here:
- ensure attestations can be deleted by setting IrI on-chain to `None`
- for any `Attestation` a, `deserialize(serialize(a)) == a`
  - test the above where `priority > 2^8-1` (want to make sure compact encoding is handled properly)
- create did, create attestation, assert exact contents of did document

Those tests can be considered out of scope for this pr so we could create a new ticket to add them.

Reply:
yeah i was thinking on what tests may be beneficial here, often its been the case to replicate some node-side tests. i'd like to also handle some expected failure cases. however these tests are extra, providing they are covered on node id also agree theyre out of scope for this PR but we should have a story to write them just so we can get the functionality merged

Reply:
https://github.com/docknetwork/sdk/issues/242